.github/workflows/ci.yml

```diff
@@ -76,6 +76,18 @@ jobs:
           cd examples/MDF
           python arrays.py -run   # test one example
 
+    - name: Build Documentation
+      run: |
+
+        pip install requests
```

Comment:
@Onoyiza Is requests used in this package? If so it should be in setup.cfg, or removed from here

Reply:
Yes, requests was used in the contributors.py file to get data from modelspec's contributors github api. I did not manually install on my local machine, it's possible that it is a direct or indirect dependency of one of the packages modelspec uses. I did pip install .[all] in the environment on my local machine and requests was one of the packages installed. 

Should I pip install all in the workflow too instead of pip install requests and pip install docs? Or maybe only pip install .[docs] and see if I do not get an import error.

Reply:
I guess it's needed to generate the docs as a whole, not just the sphinx step, so best add it here: 
https://github.com/Onoyiza/modelspec/blob/2ffa5bf4fdd7541d2124946f2f47e51965b47b4f/setup.cfg#L69